.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: bpfci
+
+on:
+  pull_request:
+
+jobs:
+  VM_Test:
+    runs-on: ubuntu-latest
+    env:
+      PROJECT_NAME: 'libbpf'
+      AUTHOR_EMAIL: "$(git log -1 --pretty=\"%aE\")"
+      KERNEL: LATEST
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: install
+        run: for f in aptitude qemu-kvm zstd binutils-dev elfutils libcap-dev libelf-dev libdw-dev python3-docutils; do sudo apt-get install $f ; done
+      - name: Kernel LATEST + selftests
+        run: export REPO_ROOT=$GITHUB_WORKSPACE ; export CI_ROOT=$REPO_ROOT/travis-ci; export VMTEST_ROOT=$CI_ROOT/vmtest ; ${CI_ROOT}/vmtest/run_vmtest.sh || exit 1
```

Comment:
random nit: REPO_ROOT seems like the only one that we really need to specify, CI_ROOT and VMTEST_ROOT are derivable and can be put into run_vmtest.sh?